tools/repos_with_ids.txt

```diff
@@ -159,3 +159,5 @@
 270|github.com/OCA/connector-spscommerce
 271|github.com/OCA/rest-framework
 272|github.com/OCA/l10n-greece
+273|github.com/OCA/oca-custom
+274|github.com/OCA/connector-telephony
```

Comment:
This one should be 228. Where do you take the number?! You have to check it on runbot itself: https://runbot.odoo-community.org/runbot/repo/github-com-oca-connector-telephony-228 (note the final number)

Reply:
@pedrobaeza done.